src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -51,7 +55,7 @@ final class ProgressBar
      * @param OutputInterface $output An OutputInterface instance
      * @param int             $max    Maximum steps (0 if unknown)
      */
-    public function __construct(OutputInterface $output, int $max = 0)
+    public function __construct(OutputInterface $output, int $max = 0, float $minSecondsBetweenRedraws = 0)
```

Comment:
The best default is not zero, but this would be behavioral change and would be caught by the tests
Since this is not technically a BC break, but more a behavioral change, we can maybe still change the default to e.g. 0.1?

Reply:
I would be happy to change it now and I did that, but on hackaton I was told I am changing too much testcases so I abandonded this attempt

Reply:
Sure sorry, my question was for more for @symfony/deciders to consider when reviewing :)